patches/net/minecraft/client/gui/screens/MenuScreens.java.patch

```diff
@@ -23,3 +23,29 @@
      }
  
      @Nullable
+@@ -58,6 +_,10 @@
+         return (MenuScreens.ScreenConstructor<T, ?>)SCREENS.get(p_96200_);
+     }
+ 
++    /**
++     * @deprecated Use {@link net.neoforged.neoforge.client.event.RegisterMenuScreensEvent} to register menu screens
++     */
++    @Deprecated
```

Comment:
Nitpick: please add a TODO comment to remove the accesstransformer for this method in the next MC version update along the lines of `TODO 1.20.5: remove AT`

Reply:
Doing it rn

Reply:
building!